### PR TITLE
Add c and c++ code to merge and fib

### PR DESCRIPTION
**Pull request by Villiamfj**
- Fibonacci changed to recursive
- Added som C and C++ benchmarks

### CODE REVIEW

File: benchmarks/fibsequence/c/bench.c [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+void stop_rapl();
+
+// test method
+long long fibb(long long a, long long b, int n) {
+    return (--n>0)?(fibb(b, a+b, n)):(a);
+}
+
+int main(int argc, char *argv[]) {
+    int fibParam = atoi(argv[1]);
+    int count = atoi(argv[2]);
```
Comment (Jakobzs) [inline benchmarks/fibsequence/c/bench.c]: Count er param 1

File: benchmarks/fibsequence/cpp/bench.cpp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    std::vector<int> v(n+1);
+    v[1] = 1;
+    transform(v.begin(), v.end()-2, v.begin()+1, v.begin()+2, std::plus<int>());
+    // "v" now contains the Fibonacci sequence from 0 up
+    return v[n];
+}
+
+int main(int argc, char *argv[]) {
+    int fib_param = std::atoi(argv[1]);
+    int count = std::atoi(argv[2]);
```
Comment (Jakobzs) [inline benchmarks/fibsequence/cpp/bench.cpp]: Count param 1

File: benchmarks/mergesort/c/bench.c [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    char* mergeParamRaw = argv[1];
+
+    // removing brackets
+    RemoveChars(argv[1], '[');
+    RemoveChars(argv[1], ']');
+
+    int* mergeParam = convertToIntArr(mergeParamRaw);
+    int mergeParamLen = sizeof(mergeParam) / sizeof(mergeParam[0]) + 1;
+
+    int count = atoi(argv[2]);
```
Comment (Jakobzs) [inline benchmarks/mergesort/c/bench.c]: Count param 1

File: benchmarks/mergesort/cpp/bench.cpp [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    // removing brackets
+    mergeParamRaw.erase(remove(mergeParamRaw.begin(), mergeParamRaw.end(), ']'), mergeParamRaw.end());
+    mergeParamRaw.erase(remove(mergeParamRaw.begin(), mergeParamRaw.end(), '['), mergeParamRaw.end());
+    
+
+    // getting numbers from mergeParamRaw
+    vector<int> mergeParam = IntVectorFromString(mergeParamRaw);
+    
+
+    int count = std::atoi(argv[2]);
```
Comment (Jakobzs) [inline benchmarks/mergesort/cpp/bench.cpp]: Count param 1

**Review by Jakobzs — CHANGES_REQUESTED** [review]

File: benchmarks/fibsequence/c/bench.c (hunk 1 above)
Comment (Villiamfj) [reply benchmarks/fibsequence/c/bench.c]: CatTypingGIF

File: benchmarks/fibsequence/cpp/bench.cpp (hunk 2 above)
Comment (Villiamfj) [reply benchmarks/fibsequence/cpp/bench.cpp]: DoneWellGIF

**Review by Jakobzs — APPROVED** [review]

**Pull request merged**